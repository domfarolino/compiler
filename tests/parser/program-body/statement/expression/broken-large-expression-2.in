program dominic is

  bool dominic;

begin

  dominic := 4 || (4 ++ (5 /* 4))

end program.
