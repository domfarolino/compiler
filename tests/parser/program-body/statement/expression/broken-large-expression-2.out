[1;31mLine 7: Error parsing expression. Expected valid syntax, but got: '|'
Line 7: Error parsing program body
[0m